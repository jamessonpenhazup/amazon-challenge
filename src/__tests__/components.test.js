import React from 'react';
import { shallow } from 'enzyme';
import BigCarousel from '../components/carousel/index';
import Footer from '../components/footer';
import Menu from '../components/menu';
import LittleCarousel from '../components/littleCarousel';
import SearchComponent from '../components/search';
describe('Testing render Components', () => {

    it('should  BigCarousel render correctly', () => {
        const wrapper = shallow( <
            BigCarousel id = {
                "movie/508"
            }
            />
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('should LittleCarousel render correctly', () => {
        const wrapper = shallow( <
            LittleCarousel id = "movie/3685" / >
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('should Footer render correctly', () => {
        const wrapper = shallow( <
            Footer / >
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('should Menu render correctly', () => {
        const wrapper = shallow( <
            Menu / >
        );
        expect(wrapper).toMatchSnapshot();
    });
    it('should SearchComponent render correctly', () => {
        const wrapper = shallow( <
            SearchComponent id = { "a" }/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});