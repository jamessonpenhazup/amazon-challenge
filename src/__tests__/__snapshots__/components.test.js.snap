// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Carousel Component should  BigCarousel render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing Carousel Component should Footer render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing Carousel Component should LittleCarousel render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing Carousel Component should Menu render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing Carousel Component should SearchComponent render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing render Components should  BigCarousel render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing render Components should Footer render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing render Components should LittleCarousel render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing render Components should Menu render correctly 1`] = `ShallowWrapper {}`;

exports[`Testing render Components should SearchComponent render correctly 1`] = `ShallowWrapper {}`;
